cirq/google/calibration/phased_fsim.py

```diff
@@ -76,3 +85,102 @@ def override_by(self, other: 'PhasedFSimParameters') -> 'PhasedFSimParameters':
             Otherwise the current values are used.
         """
         return other.merge_with(self)
+
+
+@json_serializable_dataclass(frozen=True)
+class FloquetPhasedFSimCalibrationOptions:
+    characterize_theta: bool
+    characterize_zeta: bool
+    characterize_chi: bool
+    characterize_gamma: bool
+    characterize_phi: bool
+
+
+# TODO: Fix json serialization (the default one doesn't work with tuples as dictionary keys).
```

Comment:
If you want to create a bug and set this to NOT_YET_SERIALIZABLE I can take this off your plate.

Reply:
Removed serialization from this PR